.buildkite/release-builds.yml

```
@@ -0,0 +1,22 @@
+# This pipeline is meant to be run via the Buildkite API, and is only used for release builds
+
+# Nodes with values to reuse in the pipeline.
+common_params:
+  # Common plugin settings to use with the `plugins` key.
+  - &common_plugins
+    - automattic/bash-cache#v1.5.0
```

[In another PR](https://github.com/woocommerce/woocommerce-android/pull/5862/files#diff-a3991ebf1475eb82acab13946ffc1eca02b43917f6d5bec3898f45c8b0b9bd53R5), we use `bash-cache` at 2.1.0. Should we do the same here?

```suggestion
    - automattic/bash-cache#2.1.0
```